src/core/UnistylesRuntime.ts

```diff
@@ -122,6 +149,41 @@ export class UnistylesRuntime {
         return ScreenOrientation.Portrait
     }
 
+    /**
+     * Get the pixel ratio
+     * @returns - The pixel ratio
+     */
+    public get pixelRatio() {
+        return this.unistylesBridge.pixelRatio
+    }
+
+    /**
+     * Get the font scale
+     * @returns - The font scale
+     */
+    public get fontScale() {
+        const fontScale = this.unistylesBridge.fontScale
+
+        return fontScale === 1.0 ? 1 : fontScale.toFixed(2)
+    }
```

Comment:
@jpudysz thanks so much for implementing this in 2.8.0 🙇‍♂️ 

Have just started playing with the new version and noticed `fontScale` was `number | string`, which prompted me to go look at the implementation.

Was the `toFixed` here to serve a specific purpose? Wondering if it'd make sense to always return a precise number and leave any rounding/clamping behaviour for userland implementations?

For my use of this value, I'd end up just wanting to `parseFloat()` this anyway.

Reply:
Oh no 😬

Native side reports float with 10 digits - that's why I though about rounding it. I didn't check the type and now it's string 🤦

I will fix it in today's release. 

Not sure if that amount of precision is required though